index.html

```diff
@@ -6,11 +6,211 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <section class="homepage-container">
+        <section class="hero-box">
+                <section class="hero-flexbox">
+                    <img src="img/AsterixetObelix.jpg"  class="AsterixetObelix" alt="Obelix holding cakes with Asterix standing next to him">
+
+                    <section class="menu-quote-container">
+
+<!-- 
+For the hamburger menu button I have chosen to use the functionality found at https://code-boxx.com/simple-responsive-pure-css-hamburger-menu/
+-->
+
+                    <!--
+                        <figure>
+                            <img
+                                src="img/menu-hamburger.svg"
+                                alt="Menu Button"
+                        </figure>
+                    --> 
+
+                        <section>
+                        <!-- (A) MENU WRAPPER -->
+                            <nav class="hamnav">
+                        
+                                <!-- (B) THE HAMBURGER -->
+    
+                                <!-- &#9776; is the HTML symbol for the “3 lines hamburger icon” -->
+                                    <span class="menugap"></span> <!-- Push the icon at the far right-->
+                                    <label for="hamburger">&#9776;</label>
+                                    <input type="checkbox" id="hamburger"/>
+
+                                <!-- (C) MENU ITEMS -->
+                                <div class="hamitems">
+                                    <a href="#">HOME</a>
+                                    <a href="#">CAKES</a>
+                                    <a href="#">ORDERING</a>
+                                    <a href="#">LESSONS</a>
+                                    <a href="#">ABOUT US</a>
+                                </div>
```

Comment:
Nice mobile nav 👍 
- Please use `<ul>` and `<li>` tags for lists of items like this. You've done really well for the desktop version of the nav - this should be no different. 
- Can you use CSS to make this mobile nav lay _on top_ of the homepage content, rather than pushing everything down when it's open?